stdlib/inspect.pyi

```diff
@@ -177,7 +178,7 @@ if sys.version_info >= (3, 11):
     @overload
     def getmembers_static(object: object, predicate: _GetMembersPredicate | None = None) -> _GetMembersReturn: ...
 
-def getmodulename(path: str) -> str | None: ...
+def getmodulename(path: PathLike[str]) -> str | None: ...
```

Comment:
It also accepts `str`. We should use `_typeshed.StrPath`

```suggestion
def getmodulename(path: StrPath) -> str | None: ...
```

Reply:
Oh oof, thanks for the review!